linux/kmflcomp/src/kmflcomp.c

```diff
@@ -1403,35 +1422,69 @@ char *find_first_match(char *path)
 }
 #endif
 
+// valid targets that should compile for linux
+char *valid_targets[] = {
+	"any","linux","desktop","windows","macosx"};
```

Comment:
Technically, we should only compile for linux with `any`, `linux`, or `desktop`. There may be a legacy case for including `windows` but should not be any case for including `macosx`.

Reply:
Yep. I should have added a comment to say that is temporary until the keyboards that work with linux are marked with `any`, `linux`, or `desktop`. I'll added a comment in the ibus-kmfl-dbus PR rather than hold this one up